storm/Region.cpp

```diff
@@ -226,16 +502,12 @@ void SRgnGetBoundingRectf(HSRGN handle, RECTF* rect) {
 
     HLOCKEDRGN lockedHandle;
     auto rgn = s_rgntable.Lock(handle, &lockedHandle, 0);
-
-    if (!rgn) {
-        s_rgntable.Unlock(lockedHandle);
-        return;
-    }
+    if (!rgn) return;
```

Comment:
Good catch. I thought it was calling `Unlock` from what I was looking at in WoW 3.3.5a, but it looks like `Unlock` is most likely the result of `MSVC` inlining and optimizing `Lock`'s behavior (which `Unlock`s if a pointer isn't obtained).